main.go

```diff
@@ -133,6 +134,13 @@ func validateAndGetEnvVars() (*environmentVariables, error) {
 		envVars.DevMode = developerMode
 	}
 
+	bindServers := os.Getenv("BIND_SERVERS")
+	if len(bindServers) == 0 {
```

Comment:
you can do the following to avoid the if / else

```code
envVars.BindServers = []string{}
if len(bindServers) > 0 {
envVars.BindServers = strings.Split(bindServers, ",")
} 
```